kasa/tests/device_fixtures.py

```diff
@@ -237,6 +237,11 @@ def parametrize(
     model_filter=BULBS_IOT_VARIABLE_TEMP,
     protocol_filter={"IOT"},
 )
+variable_temp_smart = parametrize(
+    "variable color temp iot",
+    model_filter=BULBS_IOT_VARIABLE_TEMP,
```

Comment:
The name says smart, but the desc and the model filter say IOT.

Reply:
Fixed